cmd/globals.go

```diff
@@ -226,6 +226,23 @@ var (
 	globalKMS crypto.KMS
 	// KMS config
 	globalKMSConfig crypto.KMSConfig
+
+	// Is compression include extensions/content-types set.
+	globalIsEnvCompression bool
+
+	// Is compression enabeld.
+	globalIsCompressionEnabled = false
+
+	// Exclude-list for compression.
+	globalCompressExtensions = []string{".txt", ".log", ".csv", ".json"}
```

Comment:
Since these are already globally defined simply use them wherever needed to have default values.

Reply:
yes ok !